### PR TITLE
EG-1021, formatted label value output, for currency and default text, tests

### PR DESCRIPTION
**Pull request by cmrhmrc**

### CODE REVIEW

File: app/models/ocelot/package.scala [hunk 1]
```diff
@@ -31,6 +32,7 @@ package object ocelot {
   def labelReferences(str: String): List[String] = plSingleGroupCaptures(labelRefRegex, str)
   def labelReference(str: String): Option[String] = plSingleGroupCaptures(labelRefRegex, str).headOption
   def isCurrency(str: String): Boolean = inputCurrencyRegex.findFirstIn(str).fold(false)(_ => true)
```
Comment (LesWilson) [inline app/models/ocelot/package.scala:34]: No test coverage for the isCurrency method

File: test/views/components/RenderTextSpec.scala [hunk 2]
```diff
@@ -36,7 +36,7 @@ class RenderTextSpec extends WordSpec with Matchers with GuiceOneAppPerSuite {
 
   trait Test {
     implicit val labels: Labels = LabelCache(Map("Blah" -> Label("Blah", Some("a value")),
-                                                                "A-Label" -> Label("A-Label", Some("33.99"))))
+                                                                "A-Label" -> Label("A-Label", Some("33.9"))))
```
Comment (LesWilson) [inline test/views/components/RenderTextSpec.scala:39]: Could you add a label that has a value >= 1000 to test commas and an exact number to test that .00 added for currency, then add a couple of tests

**Review by LesWilson — CHANGES_REQUESTED** [review]
Minor test updates

File: app/models/ocelot/package.scala (hunk 1 above)
Comment (cmrhmrc) [reply app/models/ocelot/package.scala:34]: isCurrency no longer used, removed

File: test/views/components/RenderTextSpec.scala (hunk 2 above)
Comment (cmrhmrc) [reply test/views/components/RenderTextSpec.scala:39]: yes, will do

**Review by LesWilson — APPROVED** [review]

**Pull request merged**